utils/src/main/scala/za/co/absa/enceladus/utils/implicits/StructFieldImplicits.scala

```diff
@@ -24,8 +24,8 @@ object StructFieldImplicits {
       Try(structField.metadata.getString(key)).toOption
     }
 
-    def getMetadataBoolean(key: String): Option[Boolean] = {
-      Try(structField.metadata.getBoolean(key)).toOption
+    def getMetadataStringAsBoolean(key: String): Option[Boolean] = {
+      Try(structField.metadata.getString(key).toBoolean).toOption
```

Comment:
Why converting to `Option`? Returning a `Try` is a good practice when errors are expected.